.changeset/gold-maps-cut.md

```diff
@@ -0,0 +1,21 @@
+---
+"@fluidframework/aqueduct": minor
+"@fluidframework/container-runtime": minor
+"@fluidframework/container-runtime-definitions": minor
+"@fluidframework/datastore": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
```

Comment:
@alexvy86 I guess we should only add package from the API is removed and not all packages which are in the PR?
cc: @tylerbutler